FAQ.md

```diff
@@ -57,4 +57,30 @@ Q: How do I make a singleton object mutable?
 A: Currently, singleton objects are immutable (i.e. frozen after creation), and it's generally considered
 a good practise to have global state immutable. If for some reasons you need mutable state inside such an
 object, use `@konan.ThreadLocal` annotation on the object. Also `konan.worker.AtomicReference` class could be
-used to store different pointers to frozen objects in a frozen object and atomically update those.
\ No newline at end of file
+used to store different pointers to frozen objects in a frozen object and atomically update those.
+
+Q: How can I compile my project against kotlin-native master?
+
+A: We release dev builds frequently, usually at least once a week. You can check the [list of available versions](https://bintray.com/jetbrains/kotlin-native-dependencies/kotlin-native-gradle-plugin). But in the case we recently fixed an issue and you want to check before a release is done, you can do:
+
+For the CLI, you can compile using gradle as stated in the README (and if you get errors, you can try to do a `./gradlew clean`):
```

Comment:
This is too long answer for FAQ format, please somewhat shorten it.

Reply:
i have changed to use `details+summary` https://github.com/dear-github/dear-github/issues/166#issuecomment-252282315
If the FAQ is planed to be consumed from GitHub, that should work. The other thing I can do is to update the gradle.md and put it there to make it smaller. But maybe it s fine already like this.